tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -693,7 +693,10 @@ private function checkDownload(
 		$this->response = HttpRequestHelper::get($url, $user, $password, $headers);
 		Assert::assertEquals(
 			200,
-			$this->response->getStatusCode()
+			$this->response->getStatusCode(),
+			"Expected status code is '200' but got '"
```

Comment:
```suggestion
			__METHOD__ . " Expected status code is '200' but got '"
```
For these status codes checks, it will be nice for the user to see what method it is in.

Reply:
fixed